pkg/virt-launcher/virtwrap/network/podinterface.go

```diff
@@ -322,6 +330,15 @@ func getPhase2Binding(vmi *v1.VirtualMachineInstance, iface *v1.Interface, netwo
 	return nil, fmt.Errorf("Not implemented")
 }
 
+func generateMacvtapVMNetworkingConfigurator(vmi *v1.VirtualMachineInstance, iface *v1.Interface, podInterfaceName string, launcherPID int) (*MacvtapBindMechanism, error) {
```

Comment:
naming on this factory functions is unrealted to what is building, what do you think about `newMacvtapBindMechanismWithLaunchID`

Reply:
I'll try to shed some light into what I'm trying to achieve in the PR's description.

Let's revisit this comment after that.